app/src/main/res/layout/admin_settings_dialog.xml

```diff
@@ -10,21 +10,31 @@
 
   <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:profile="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
 
-    <org.oppia.android.app.profile.ProfileInputView
+    <com.google.android.material.textfield.TextInputLayout
       android:id="@+id/input_pin"
+      style="@style/TextInputLayout"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:layout_marginStart="24dp"
       android:layout_marginTop="32dp"
       android:layout_marginEnd="24dp"
+      android:hint="@string/admin_auth_pin"
       android:layout_marginBottom="20dp"
```

Comment:
Is hint is correct?

Reply:
Yes it is,
This is for Admin Pin after submitting this the reset pin dialog appears
please check the screenshots for reference

Reply:
previously we are using `app:label="@string/admin_settings_label"`
Let me know If I am mistaking something here!!

Reply:
I'm sorry i was litter confused here as the value for both string were the same
I've changed it now